contracts/oracle/FastGasOracle.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../Interfaces/IOracle.sol";
+import "../Interfaces/IChainlinkOracle.sol";
+import "../lib/LibMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "prb-math/contracts/PRBMathUD60x18.sol";
+
+/**
+ * @dev The following is a sample Gas Price Oracle Implementation for a Liquidation Oracle.
+ *      It references the Chainlink fast gas price and returns the fast price of gas in wei.
+ */
+contract FastGasOracle is IOracle, Ownable {
```

Comment:
You don't need to make a new contract for this. You can just use `contracts/oracle/Oracle.sol`, which is already done here: https://github.com/tracer-protocol/tracer-protocol/blob/contracts-v2/deploy/FullDeployTest.js#L55